.env.ci

```diff
@@ -14,7 +14,7 @@
 # https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
 
 ###> symfony/framework-bundle ###
-APP_ENV=dev
+APP_ENV=test
```

Comment:
This seems to have implications for the project setup in the readme?

Reply:
Sharp! Updated the README.md